### ISSUE TITLE
Can't build: Argument labels '(dynamicProvider:)' do not match any available overloads

### ISSUE THREAD

**Issue by n8henrie** [ColdGrub1384/Pyto#68]
First-timer and new to swift. I enjoy Pythonista and wanted to give Pyto a shot as it's open source, but I'm having trouble building.

Ran the setup script. Xcode 10.3, MacOS 10.14.6, building from `master`, getting `Argument labels '(dynamicProvider:)' do not match any available overloads` and then several: `Pyto/Pyto/Python Bridging/UI/Other/PyColor.swift:94:37: Type 'UIColor' has no member 'label'`

Also tried building from the `10.0` tag, but was getting an error about an OpenSSL github repo that seems to have disappeared (bruceyibin/OpenSSL.git).

Would appreciate any pointers, TIA!

**Comment by ColdGrub1384** [comment]
It should work with Xcode 11 beta. `(dynamicProvider:)` is for iOS 13. If you build with Xcode 11, it will also work on iOS 12.

**Comment by n8henrie** [comment]
Hm, I tried with Xcode 11, building for iOS 12, (don't have any iOS 13 beta devices at the moment) but didn't have any luck; lots of errors about my developer certificate. I've been able to build and locally install other open source iOS apps (though not lately).

Unfortunately, I misread your response and thought the issue must be due to not having iOS 13, so I deleted the Xcode beta. I may give this another shot soon, but I'll close the issue for now -- I live in a very rural town with poor internet, and it will probably take me another several days to re-download Xcode 11.

Thanks again for your kind response, and for an interesting project.
